### PR TITLE
feat(sd_jwt): ✨ statement to verify if a set of disclosures is contained in a sd_jwt

### PR DESCRIPTION
**Pull request by matteo-cristino**

### CODE REVIEW

**Comment by matteo-cristino** [comment]
@jaromil I'm not so sure about the "contains" word, any idea on that?

File: src/lua/zencode_sd_jwt.lua [hunk 1]
```diff
@@ -416,6 +416,21 @@ When("use signed selective disclosure '' only with disclosures ''", function(ssd
     ssd.disclosures = disclosure
 end)
 
+IfWhen("verify selective disclosure '' contains disclosures ''", function(ssd_name, lis)
```
Comment (jaromil) [inline src/lua/zencode_sd_jwt.lua:419]: more aligned with other statements to use:

`is found` or `are found`

`verify disclosures '' are found in selective disclosure ''`

perhaps we can make `is` and `are` equivalent

File: src/lua/zencode_sd_jwt.lua [hunk 2]
```diff
@@ -416,6 +416,21 @@ When("use signed selective disclosure '' only with disclosures ''", function(ssd
...
+    local ssd = have(ssd_name)
+    local disclosed_keys = have(lis)
+    for _,k in pairs(disclosed_keys) do
```
Comment (jaromil) [inline src/lua/zencode_sd_jwt.lua:422]: check that this is an array else pairs will act weird

also accept case in which there is a single one being verified fo be found.

**Review by jaromil — CHANGES_REQUESTED** [review]

File: src/lua/zencode_sd_jwt.lua (hunk 1 above)
Comment (matteo-cristino) [reply src/lua/zencode_sd_jwt.lua]: Used `are found`

File: src/lua/zencode_sd_jwt.lua (hunk 2 above)
Comment (matteo-cristino) [reply src/lua/zencode_sd_jwt.lua:427]: added checks and possibility to have a single element

**Review by jaromil — APPROVED** [review]

**Comment by jaromil** [comment]
Well done. Later on we may want to  `gsub(tt, ' are ', ' is ')` in the zencode parser, but will deserve another PR.

**Pull request merged**